featurebook/04 - My page and related.md

```diff
@@ -58,3 +58,20 @@ Thanks to this feature, we enable near real time communication capabilities. I.e
 * For 2/, for most cases, we should be covered by 1/. And there is also a timer that periodically checks that we are still on "My page". And if not => redirect.
 
 </details>
+
+
+`@Scenario:`
+
+## WHEN a new note or message is added to a thread the user is watching, ...
```

Comment:
Poti reformula dupa modelul anterioarei (pentru uniformitate):
 GIVEN installed as PWA, 
Si sa fie mai succint, fara a intra in detalii:
 `When a new unread note / forum post` THEN
sau chiar 
`When a a new unread message THEN`
Poti lasa doar 
`THEN a notification is displayed` si sa pui o imagine in loc de precizarea (`via ...` ) precizam deja in given ca suntem in cazul PWA

Reply:
Done